public.env

```diff
@@ -6,7 +6,7 @@ PH_DB_USER=phenopolis_api
 PH_DB_PASSWORD=phenopolis_api
 PH_DB_PORT=5432
 
-S3_VCF_FILE_URL="s3://phenopolis-vcf/August2019/merged2.vcf.gz"
+S3_VCF_FILE_URL="schema/small_demo.vcf.gz"
```

Comment:
IMO - we should rename this environment variable so that it is not specific to S3 

(And then add a comment in the Readme etc. that this variable can be an S3 link (provided Secret key etc. variables are provided)

Reply:
Good point. I'll do this and then merge.

Reply:
If you do change the variable - let me know before you merge - so that I can add this variable to the AWS EB environment...